app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -635,11 +635,11 @@ def outright_sale?
   end
 
   def joint_purchase?
-    field_114 == 1
+    field_15 == 1
   end
```

Comment:
Looks like this method is wrong in 2023 as well?

Reply:
Huh, you're right - I'd assumed it had changed for 2024.